src/Order.Data/OrderContext.cs

```diff
@@ -1,160 +1,208 @@
+                    .WithMany(p => p.Items)
+                    .HasForeignKey(d => d.OrderId)
+                    .OnDelete(DeleteBehavior.ClientSetNull)
+                    .HasConstraintName("order_item_oifk_1");
+
+                entity.HasOne(d => d.Product)
+                    .WithMany(p => p.OrderItem)
+                    .HasForeignKey(d => d.ProductId)
+                    .OnDelete(DeleteBehavior.ClientSetNull)
+                    .HasConstraintName("order_product_oifk_1");
+
+                entity.HasOne(d => d.Service)
+                    .WithMany(p => p.OrderItem)
+                    .HasForeignKey(d => d.ServiceId)
+                    .OnDelete(DeleteBehavior.ClientSetNull)
+                    .HasConstraintName("order_service_oifk_1");
+            });
+
+            modelBuilder.Entity<OrderProduct>(entity =>
+            {
+                entity.ToTable("order_product");
+
+                entity.HasIndex(e => e.ServiceId)
+                    .HasName("order_service_opfk_1");
+
+                entity.Property(e => e.Id).HasColumnType("binary(16)");
+
+                entity.Property(e => e.Name)
+                    .IsRequired()
+                    .HasMaxLength(100)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.ServiceId)
+                    .IsRequired()
+                    .HasColumnType("binary(16)");
+
+                entity.HasOne(d => d.Service)
+                    .WithMany(p => p.OrderProduct)
+                    .HasForeignKey(d => d.ServiceId)
+                    .OnDelete(DeleteBehavior.ClientSetNull)
+                    .HasConstraintName("order_service_opfk_1");
+            });
+
+            modelBuilder.Entity<OrderService>(entity =>
+            {
+                entity.ToTable("order_service");
+
+                entity.Property(e => e.Id).HasColumnType("binary(16)");
+
+                entity.Property(e => e.Name)
+                    .IsRequired()
+                    .HasMaxLength(100)
+                    .IsUnicode(false);
+            });
+
+            modelBuilder.Entity<OrderStatus>(entity =>
+            {
+                entity.ToTable("order_status");
+
+                entity.HasIndex(e => e.Name)
+                    .HasName("Status");
+
+                entity.Property(e => e.Id).HasColumnType("binary(16)");
+
+                entity.Property(e => e.Name)
+                    .IsRequired()
+                    .HasMaxLength(20)
+                    .IsUnicode(false);
+            });
+        }
+
+        /// <summary>
+        /// This configuration is required because SQLite provider cannot apply aggregate operators in queries 
+        /// for navigation properties with decimal and DateTimeOffset type.
+        /// SQLite provider is used only for tests, so this change applies only for tests.
+        /// </summary>
+        private void ConfigureConversionsForTestDatabase(ModelBuilder modelBuilder)
```

Comment:
This might probably contain also conversions for Guid ids. With that we could get rid of method bellow and also creating of different queries for production code and tests.